Sources/CrowdNotifierSDK/Crypto/CryptoUtils.swift

```diff
@@ -186,26 +187,27 @@ final class CryptoUtils {
     }
 
     public static func generateIdentityV3(startOfInterval: Int, qrCodePayload: Bytes) -> Bytes? {
-        guard let (nonce1, nonce2, _) = CryptoUtilsBase.getNoncesAndNotificationKey(qrCodePayload: qrCodePayload) else {
+        guard let (noncePreId, nonceTimekey, _) = CryptoUtilsBase.getNoncesAndNotificationKey(qrCodePayload: qrCodePayload) else {
             return nil
         }
 
-        guard let preid = crypto_hash_sha256(input: DomainKeys.preid.bytes + qrCodePayload + nonce1) else {
+        let intervalLength = Int32(bigEndian: 3600) // at the moment, hour buckets are used
```

Comment:
If this is ever going to change, I'd put it as an argument to the method.

Reply:
I think at some point we can add a configuration option for the duration to the SDK. Before that, it's probably not worth the effort to refactor this.